playgrounds/vue/components/examples.vue

```diff
@@ -0,0 +1,69 @@
+<template>
```

Comment:
Looks like it's app.vue which is moved and renamed. If so, please use git move

Reply:
Hi, same goes here I modified the file (moved all examples in a separate folder) and renamed it.